Sources/SwiftProtobuf/Google_Protobuf_Duration+Extensions.swift

```diff
@@ -130,18 +132,41 @@ extension Google_Protobuf_Duration: _CustomJSONCodable {
     }
 }
 
-
 extension Google_Protobuf_Duration: ExpressibleByFloatLiteral {
     public typealias FloatLiteralType = Double
 
     public init(floatLiteral value: Double) {
+        // TODO: this rounds the nanos part towards zero, not to the nearest
```

Comment:
So you've typed ten 9's. That is an order of magnitude closer to 2.0 than it is to 1.999999999 (with nine 9's), so I am advocating that it should result in (2, 0).